src/data/documentation-links.yaml

```diff
@@ -0,0 +1,149 @@
+- id: Getting started
```

Comment:
@corwintines this list is hardcoded, if we add/remove a doc or edit a path we must remember to keep this list updated which is not ideal.

Reply:
Porting over discord discussion here, the reason I opted for this was build the structure for the docs navbar

Reply:
This is a similar approach to what is used on ethereum.org